modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -2389,7 +2389,7 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
       );
     }
 
-    if (impliedForwarderVersion === 0 || impliedForwarderVersion === 3) {
+    if (impliedForwarderVersion === 0 || impliedForwarderVersion === 3 || impliedForwarderVersion === 6) {
```

Comment:
can you create a const or enums for `evm walletVersions` that support forwarder addresses. This functionality is not well documented and makes it hard for others to parse through and understand.

Reply:
The forwarder version is smart contract-based, unlike versions 3 and 5, which use MPC-derived addresses. Creating an enum just for versions 3 and 5 seems excessive to me, especially since we're completely deprecating version 3.